src/drawscene.cpp

```diff
@@ -529,8 +566,13 @@ void draw_scene(video::IVideoDriver *driver, scene::ISceneManager *smgr,
 		show_hud = false;
 	}
 	else {
-		draw_plain(camera, show_hud, hud, driver,
-				draw_wield_tool, client, guienv);
+		int us = g_settings->getU16("undersampling");
+		if(us)
+			draw_undersampled(camera, show_hud, hud, driver,
+					smgr, screensize, draw_wield_tool, client, guienv, skycolor, us);
```

Comment:
Done. Checks configuration inside the `draw_plain` function, just as others check for parallax settings.